packages/core/src/DdSdkReactNativeConfiguration.tsx

```diff
@@ -267,6 +267,30 @@ export class DdSdkReactNativeConfiguration {
      */
     public bundleLogsWithTraces: boolean = DEFAULTS.bundleLogsWithTraces;
 
+    /**
+     * Enables tracking of non-fatal ANRs on Android.
+     * By default, the reporting of non-fatal ANRs on Android 30+ is disabled because it would
+     * create too much noise over fatal ANRs. On Android 29 and below, however,
+     * the reporting of non-fatal ANRs is enabled by default,
+     * as fatal ANRs cannot be reported on those versions.
+     */
+    public trackNonFatalAnrs?: boolean;
+
+    /**
+     * The app hang threshold in milliseconds for non-fatal app hangs on iOS.
```

Comment:
Also noted as I'm implementing this myself. This is app hang time in seconds, not milliseconds.